core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```
@@ -1,3 +1,132 @@
+<ng-template #TranslationEditorTourStep>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)